### PR TITLE
ECO-561 remove need of activate

### PR DESCRIPTION
**Pull request by doodyparizada**
Remove the need to activate the account before using the SDK.

### CODE REVIEW

File: scripts/src/public/routes/index.ts [hunk 1]
```diff
@@ -63,8 +63,10 @@ function Router(): ExtendedRouter {
 						return (target as any)[name](path, async (req: express.Request, res: express.Response, next: express.NextFunction) => {
 							const token = await authenticate(req);
 							const user = await db.User.findOneById(token.userId);
-							// XXX scopes should be per token and should not consider user data
-							if (scopes.includes(AuthScopes.TOS) && (!user || !user.activated || token.createdDate < user.activatedDate!)) {
+
+							if (!user) {
+								// This error now defines an inconsistent state in the DB where a token exists but not user is found
+								// This should never happen as the token.user_id is a foreign key to the users table
 								throw TOSMissingOrOldToken();
```
Comment (YohayBar) [inline scripts/src/public/routes/index.ts]: maybe change the exception to UserWasNotFound or something like that

File: scripts/src/public/routes/index.ts [hunk 2]
```diff
@@ -63,8 +63,10 @@ function Router(): ExtendedRouter {
 						return (target as any)[name](path, async (req: express.Request, res: express.Response, next: express.NextFunction) => {
 							const token = await authenticate(req);
 							const user = await db.User.findOneById(token.userId);
-							// XXX scopes should be per token and should not consider user data
-							if (scopes.includes(AuthScopes.TOS) && (!user || !user.activated || token.createdDate < user.activatedDate!)) {
```
Comment (doodyparizada) [inline scripts/src/public/routes/index.ts]: remove the extra token from the activate method

**Comment by YohayBar** [comment]
@doodyparizada Do you also change the auth token and remove activated or you set activated true when returning the the first token?

File: scripts/src/public/routes/index.ts (hunk 1 above)
Comment (liorama) [reply scripts/src/public/routes/index.ts]: 👍

**Comment by liorama** [comment]
You need to remove the `didNotApproveTOS` system test

**Pull request merged**